e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -523,23 +516,20 @@ describeEE("scenarios > embedding > full app", () => {
         },
       });
 
+      cy.get("@postMessage").invoke("resetHistory");
       cy.findByTestId("app-bar").findByText("Our analytics").click();
 
       cy.findByRole("heading", { name: "Metabase analytics" }).should(
         "be.visible",
       );
-      cy.get("@postMessage").then(postMessage => {
-        expect(
-          postMessage.lastCall.calledWith({
-            metabase: {
-              type: "frame",
-              frame: {
-                mode: "fit",
-                height: 800,
-              },
-            },
-          }),
-        ).to.be.true;
+      cy.get("@postMessage").should("have.been.calledWith", {
```

Comment:
Change this because it seems to always fail when stress testing. I don't know why.
- [PR stress tests](https://github.com/metabase/metabase/actions/runs/9368575231)
- [Master stress tests](https://github.com/metabase/metabase/actions/runs/9368577269)